commands/firetrade.js

```diff
@@ -1,109 +1,162 @@
-const moment = require("moment");
+const https = require("https")
+const async = require('async');
 
 /**
- * Displays Fire Trader's price for an item.
+ * Displays item buy prices for EQ traders.
  *
- * @example   !firetrading <item name>
+ * @example   !tr[ade] <item name>
  */
 exports.run = async (client, message, args, level) => { // eslint-disable-line no-unused-vars
-  const itemList = client.tornData.items;
-  const lordofFire = client.quotedMembers.get('lordoffire');
-  const timestamp = moment().format();
-
-  /**
-   * Returns an embed object with Fire Trader's pricing for an item.
-   *
-   * @param   {Object}   item   The Torn item object.
-   * @param   {String}   id     The Torn item ID.
-   * @return  {Object}   A Discord embed object.
-   */
-  function embedFireTrading(item, id) {
-    return {
-      'embed': {
-        'color': client.config.colors.lof,
-        'title': 'LordofFire\'s price:',
-        'description': '[Start Trade](https://www.torn.com/trade.php#step=start&userID=2411517/) ' +
-          '| [Pricelist](https://tinyurl.com/fire-trading-pricelist/)',
-        'thumbnail': {
-          'url': 'https://alltornup.netlify.com/TornItems965/' + id + '.png'
-        },
-        'author': {
-          'name': item.name + ' #' + id
-        },
-        'image': {
-          'url': 'https://arsonwarehouse.com/!/lordoffire/bids/' + id + '@2x.png?timestamp=' + timestamp
-        },
-        'footer': {
-          'text': 'Fire Trading'
-        }
-      }
-    };
-  }
+  const item_list_cache = client.tornData.items;
+  const traders = client.config.params.firetrade.traders;
```

Comment:
is the `firetrade` prop name a holdover to the old naming? I would think it should just be `client.config.params.traders`